eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/client/impl/PubClientImpl.java

```diff
@@ -73,7 +73,7 @@ public void close() {
             task.cancel(false);
             super.close();
         } catch (Exception e) {
-            e.printStackTrace();
+            log.error("PubClientImpl|{}|stopped!", clientNo, e);
         }
```

Comment:
The logger here is in a catch block so the log message would better be:

```suggestion
        } catch (Exception e) {
            log.error("PubClientImpl|{}|close failed!", clientNo, e);
        }
```